flask_resources/context.py

```diff
@@ -58,6 +60,15 @@ def __exit__(self, type, value, traceback):
         """Pop the resource context manager from the current request."""
         del g.resource_requestctx
 
+    def inject_original_request(self):
+        """."""
+        self.original_request = {
+            'accept_mimetypes': request.accept_mimetypes,
+            'args': request.args,
+            'content_type': request.content_type,
+            'data': request.data,
+        }
+
```

Comment:
Major: Like Pablo said, there is no reason to have resource_requestctx if we keep `request` around. But we do want to make this separation because it gives us huge flexibility, separation of concerns and guarantees. Also this is called in the `__init__` when/where we are not necessarily in a request context.